lib/tasks/populate_registrant_id.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+namespace :registrant_id do
+  desc 'Backfills all registration.registrant_id values with their WCIF-generated registrant_id'
+  task :populate do
```

Comment:
Also, this currently doesn't call the Rails classloader, meaning all models are not autoloaded. I wonder how this ever worked on your machine :stuck_out_tongue_winking_eye:

Reply:
That's probably it. Athough I'm not too knowledgeable on the specifics of Spring, but it seems like that's the only possible explanation